include/glibby/primitives/pointND.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <cmath>
+
+namespace glibby
+{
+	template<typename T, int N>
+	struct PointND
+	{
+		int axis = N;
+		T points[N];
+	};
+
```

Comment:
can you delete the point2D struct and add a typedef here? there are some examples using the vec struct in math/general_math.h

Reply:
while you're at it, can you rename this file as just "point.h" and delete the point2D.h file